CRM/Utils/Token.php

```diff
@@ -1852,7 +1852,7 @@ public static function legacyContactTokens() {
    */
   public static function getCustomFieldTokens($entity, $usedForTokenWidget = FALSE) {
     $customTokens = [];
-    $tokenName = $usedForTokenWidget ? "{contribution.custom_%d}" : "custom_%d";
+    $tokenName = $usedForTokenWidget ? '{' . strtolower($entity) . '.custom_%d}' : 'custom_%d';
```

Comment:
Could you use `mb_strtolower` instead? It tends to be more reliable for when running CiviCRM in non-English (well, more specifically when running in Turkish, example: https://github.com/civicrm/civicrm-packages/pull/324).